src/main/java/org/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -179,7 +181,29 @@ public void keyReleased(java.awt.event.KeyEvent e) {
             public void keyPressed(java.awt.event.KeyEvent e) {
                 //We need to consume this event here to prevent the propgation of keybinding events back to the JFrame
 
-                e.consume();
+                Optional<KeyCode> keyCode = Arrays.stream(KeyCode.values()).filter(k -> k.getName().equals(e.getKeyText(e.getKeyCode()))).findFirst();
```

Comment:
Can you please encapsulate this code in a `Optional<KeyBinding> KeyPrefs#mapToKeyBinding` method that accepts a `awt.KeyEvent`.